### PR TITLE
Load clients into the race with custom gamemode/player type

### PR DESCRIPTION
**Pull request by vabold**

### CODE REVIEW

**Comment by vabold** [comment]
Can someone review the ASM and make sure the patch macros are correct?

File: payload/game/kart/KartObject.S [hunk 1]
```diff
@@ -1,5 +1,22 @@
 #include <Common.S>
 
+PATCH_BL_START(KartObject_Create, 0x34)
```
Comment (Gaberboo) [inline payload/game/kart/KartObject.S:3]: Since you don't use LR, PATCH_B makes more sense. All other patch macros are good.

Comment (vabold) [reply payload/game/kart/KartObject.S]: Resolved in https://github.com/stblr/mkw-sp/commit/60a4da2d8f12f2944134fa55299f2491470a7850.

**Comment by vabold** [comment]
@stblr implemented it separately into the "rollback" branch.

**Pull request closed without merging**